admin/workspace-providers/deployment.md

```diff
@@ -158,8 +158,7 @@ values changes; you can do this with
 helm upgrade coder-workspace-provider coder/workspace-provider \
     --version=[CODER_VERSION] \
     --atomic \
-    --install \
-    --force
+    --install
```

Comment:
We no longer use `--force` for our internal deploys (preview, kind, dogfood, workspace providers) so I think it makes sense to update the docs accordingly, yeah.